Packs/CommonScripts/Scripts/ZipFile/ZipFile.py

```diff
@@ -4,13 +4,36 @@
 
 import re
 import shutil
-import zipfile
-import pyminizip
+import pyzipper
 from os.path import isfile
 
 ESCAPE_CHARACTERS = r'[/\<>"|?*]'
 
 
+def unzip(zip_file_path: str, password: str = None):
+    with pyzipper.AESZipFile(zip_file_path) as zf:
+        zf.pwd = bytes(password, 'utf-8') if password else None
+        zf.extractall()
+
+
+def compress_multiple(file_names: List[str], zip_name: str, password: str = None):
+    compression = pyzipper.ZIP_DEFLATED
+    encryption = pyzipper.WZ_AES if password else None
```

Comment:
why chose this encryptian? Is this different than the 1 we were using before? Could this have any negative impact?

Reply:
After some research, this is the standard way of encrypting a ZipFile:

https://stackoverflow.com/questions/24610452/what-is-the-encryption-algorithm-used-by-zip-files

https://kb.winzip.com/help/help_encryption.htm?alid=322760552.1716195913

It is a better encryption method than the previous lib used.